.github/workflows/publish_docker_images_cron.yml

```diff
@@ -2,7 +2,7 @@ name: Schedule Build Docker Images For Main Images And Nightly
 
 on:
   schedule:
-    - cron: '0 0 * * *'
+    - cron: '*/10 * * * *'
```

Comment:
that would run at every 10th minute, is that really what we want ?

https://crontab.guru/#*/10_*_*_*_*

Reply:
Fixed and changed to execute at 00:30 everyday